internal/cli/atlas/kubernetes/config/generate.go

```diff
@@ -110,11 +133,13 @@ func GenerateBuilder() *cobra.Command {
 		},
 	}
 
+	supportedVersions := features.GetSupportedVersions()
 	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.ProjectID)
 	cmd.Flags().StringVar(&opts.OrgID, flag.OrgID, "", usage.OrgID)
 	cmd.Flags().StringSliceVar(&opts.clusterName, flag.ClusterName, []string{}, usage.ExporterClusterName)
 	cmd.Flags().BoolVar(&opts.includeSecrets, flag.OperatorIncludeSecrets, false, usage.OperatorIncludeSecrets)
 	cmd.Flags().StringVar(&opts.targetNamespace, flag.OperatorTargetNamespace, "", usage.OperatorTargetNamespace)
+	cmd.Flags().StringVar(&opts.operatorVersion, flag.OperatorVersion, supportedVersions[len(supportedVersions)-1], usage.OperatorVersion)
```

Comment:
looking at how `features.GetSupportedVersions()` is implemented I have some mild concerns always defaulting to the newest CRD may be risky not familiar enough with the CRD ecosystem to have a strong opinion but in the CLI we aim for the "least surprise" approach, meaning behavior of a command should not change between versions and the current approach lands in the "it'll change at random point" category

Reply:
I understand your concern. But I'd expect a new version of the CLI to generate an output for a newer version of the operator. One possible solution I can think of is that I can make this flag `required` without a default version. That would force one to explicitly set this flag.

Reply:
We chatted this over. I'd like to go ahead with it as is. The worst case is that the Operator return to the user that they're trying to import something newer than the operator itself. We may iterate on this later to try and read from Kubernetes the actual version of the operator in use.